### ISSUE TITLE
MultiSelect: NVDA/JAWS should announce the combo box (multiselect) items are checked or unchecked when it receives keyboard focus.

### ISSUE THREAD

**Issue by previta21** [primefaces/primeng#11739]
### Describe the bug

When using Multiselect component NVDA/JAWS should announce the combo box items are checked or unchecked when it receives keyboard focus.
The expected output was achieved with the earlier version of Primeng. The current version does not have html input element inside div element. 
Example
(
<input type=” checkbox”>
)

### Environment

Production

### Reproducer

https://www.primefaces.org/primeng/multiselect

### Angular version

13.3.1

### PrimeNG version

13.3.0

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

v14.18.3

### Browser(s)

All

### Steps to reproduce the behavior

We have verified the issue on below Primeng URL
URL: (https://www.primefaces.org/primeng/multiselect)

Steps to reproduce:
1.Start NVDA, once the above URL is open.
2. Also, add virtualScroll="true" inside p-multiselect so that the NVDA will start reading the data from the multiselect.
3. To navigate between the multiselect items use alt+down arrow key.
4. When we click on enter to check/uncheck the checkbox NVDA/JAWS does not read if the checkbox is checked or unchecked.

### Expected behavior

When using Multiselect component NVDA/JAWS should announce the combo box items are checked or unchecked when it receives keyboard focus.

**Comment by cetincakiroglu** [comment]
Hi,

Accessibility is a major concern of the Prime UI libraries and PrimeNG is no exception. [PrimeTek](https://www.primetek.com.tr/) teams have initiated a significant process to review and enhance the accessibility features of the components. This guide documents the foundation of the general guidelines that PrimeNG will follow and each component documentation will have a separate Accessibility section that states the keyboard support, screen reader compatibility, the implementation details along with tips to achieve WCAG compliancy. This work has been initiated in Q2 2022 and planned to be completed by early Q3. PrimeNG will be the reference implementation which then will be ported to PrimeFaces, PrimeNG and PrimeVue.

**Comment by 12-shweta** [comment]
@cetincakiroglu is this bug is fixed?

**Comment by msftedad** [comment]
Hi Team,
any update on this ticket??

**Comment by msftedad** [comment]
Hi Team
Is this bug is fixed? Please confirm

**Comment by msftedad** [comment]
Hi @cetincakiroglu ,This issue is still reproducible. Can you please reopen this issue. Thanks

**Comment by cetincakiroglu** [comment]
Hi,

We've recently release an accessibility update for the multiselect, could you please check it with v16.7.0

**Comment by msftedad** [comment]
Hi @cetincakiroglu, Verified this issue on latest version 17.6.7.1 but issue is still repro'ing. Can you please reopen this issue. Thanks

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by msftedad** [comment]
Hi team, Any update on this issue?

**Comment by mehmetcetin01140** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
